assets/extensions/extensions.scss

```diff
@@ -242,6 +243,7 @@ $wordpress-break: 961px;
 				padding-left: 20px;
 				flex-shrink: 0;
 				flex-direction: row-reverse;
+				margin-left: auto;
```

Comment:
I didn't see a difference when this margin is applied. Does it fix something? Or maybe it was to fix some old styling issue that was fixed by other PRs?

Reply:
Yep these are all fixes that were made obsolete from other PRs. Forgot to check them when rebased a73dfe5fec5a02b490a2f728ffaa11239586f587